astropy/utils/console.py

```diff
@@ -691,7 +691,8 @@ def _silent_update(self, value=None):
         pass
 
     @classmethod
-    def map(cls, function, items, multiprocess=False, file=None, step=100):
+    def map(cls, function, items, multiprocess=False, ipython_widget=False,
```

Comment:
Unless there is a good reason for the opposite, the order of kwargs should be kept and new ones added to the end.